src/api/content-rest-api/api/categories.api.ts

```diff
@@ -0,0 +1,496 @@
+    */
+    getCategory(categoryId: string, opts?: any): Promise<CategoryEntry> {
+        throwIfNotDefined(categoryId, 'categoryId');
+        opts = opts || {};
+        const postBody: null = null;
+
+        const pathParams = {
+            'categoryId': categoryId
+        };
+
+        const queryParams = {
+            'fields': buildCollectionParam(opts['fields'], 'csv'),
+            'include': buildCollectionParam(opts['include'], 'csv')
+        };
+
+        const headerParams = {};
+        const formParams = {};
+
+        const contentTypes = ['application/json'];
+        const accepts = ['application/json'];
+
+        return this.apiClient.callApi(
+            '/category/{categoryId}', 'GET',
+            pathParams, queryParams, headerParams, formParams, postBody,
+            contentTypes, accepts , CategoryEntry);
+    }
+
+    /**
+        * List of categories that node is assigned to
+        *
+        * Gets a list of categories for node with nodeId.
+
+        * @param nodeId The identifier of a node.
+        * @param opts Optional parameters
+        * @param opts.skipCount The number of entities that exist in the collection before those included in this list.
+    If not supplied then the default value is 0.
+    (default to 0)
+        * @param opts.maxItems The maximum number of items to return in the list.
+    If not supplied then the default value is 100.
+    (default to 100)
+        * @param opts.fields A list of field names.
+
+    You can use this parameter to restrict the fields
+    returned within a response if, for example, you want to save on overall bandwidth.
+
+    The list applies to a returned individual
+    entity or entries within a collection.
+
+        * @return Promise<CategoryPaging>
+    */
+    getCategoryLinksForNode(nodeId: string, opts?: any): Promise<CategoryPaging> {
+        throwIfNotDefined(nodeId, 'nodeId');
+        opts = opts || {};
+        const postBody: null = null;
+
+        const pathParams = {
+            'nodeId': nodeId
+        };
+
+        const queryParams = {
+            'skipCount': opts['skipCount'],
+            'maxItems': opts['maxItems'],
+            'fields': buildCollectionParam(opts['fields'], 'csv')
+        };
+
+        const headerParams = {};
+        const formParams = {};
+
+        const contentTypes = ['application/json'];
+        const accepts = ['application/json'];
+
+        return this.apiClient.callApi(
+            '/nodes/{nodeId}/category-links', 'GET',
+            pathParams, queryParams, headerParams, formParams, postBody,
+            contentTypes, accepts , CategoryPaging);
+    }
+
+    /**
+    * Deletes the category **categoryId**.
+    *
+    * This will cause everything to be removed from the category.
+    * You must have admin rights to delete a category.
+    *
+    * @param categoryId The identifier of a category.
+    * @return Promise<{}>
+    */
+    deleteCategory(categoryId: string): Promise<any> {
```

Comment:
Did you use "any" because it returns nothing? If yes then maybe better to use void?

Reply:
Right I missed that, changed